imageio/core/v3_plugin_api.py

```diff
@@ -20,6 +20,8 @@ class ImageProperties:
         The shape of the loaded ndimage.
     dtype : np.dtype
         The dtype of the loaded ndimage.
+    n_images : int or None
```

Comment:
```suggestion
    n_images : int
```

We don't document `None` if it is the default value.